### ISSUE TITLE
An equation on optimize.html not rendered properly

### ISSUE THREAD

**Issue by heytitle** [scipy/scipy#7784]
An equation in `Unconstrained minimization of multivariate scalar functions` section on `optmize.html` is not rendered properly.

### Reproducing code example:
https://docs.scipy.org/doc/scipy-0.19.1/reference/tutorial/optimize.html#unconstrained-minimization-of-multivariate-scalar-functions-minimize
![image](https://user-images.githubusercontent.com/1214890/29584833-45c16aca-8785-11e7-9597-3710258101c5.png)
### Error message:
```
f\left(\mathbf{x}\right)=\sum_{i=1}^{N-1}100\left(x_{i}-x_{i-1}^{2}\right)^{2}+\left(1-x_{i-1}\right)^{2}.
```

### Scipy/Numpy/Python version information:
```
Scipy version scipy-0.19.1
```

**Comment by cdouglass** [comment]
I couldn't reproduce this locally on master but am unsure whether this is because it's fixed there or something else about my setup is different.

MathJax is failing to enter a `displaymath` environment both there and in a couple other parts of that page:
![hessian](https://user-images.githubusercontent.com/8453868/29918131-24fd208e-8df9-11e7-8f7d-18404e039632.png)
![conjugate_gradient](https://user-images.githubusercontent.com/8453868/29918139-2c9160bc-8df9-11e7-85bd-f0687bf4f0f0.png)

Speculation: I wonder if something changed in #7376 when we switched from CDN to bundled MathJax?

Found some maybe relevant context at #4595, #4647.

**Comment by rgommers** [comment]
I've checked and it is fixed in master (I built the 0.19.1 docs with the same setup). One of gh-4595 or gh-4647 must have fixed it, they indeed look relevant.

Thanks @heytitle and @cdouglass. Not updating the 0.19.1 docs for this anymore, will be fixed with the next release. So closing.
